ground/src/main/java/com/google/android/ground/persistence/sync/SurveySyncWorker.kt

```diff
@@ -20,9 +20,10 @@ import android.content.Context
 import androidx.hilt.work.HiltWorker
 import androidx.work.CoroutineWorker
 import androidx.work.Data
+import androidx.work.ListenableWorker.Result.failure
+import androidx.work.ListenableWorker.Result.retry
+import androidx.work.ListenableWorker.Result.success
 import androidx.work.WorkerParameters
-import com.google.android.ground.Config.MAX_SYNC_WORKER_RETRY_ATTEMPTS
```

Comment:
Can we cleanup this var from `Config.kt`?

Reply:
Done!